### DISCUSSION TITLE
Support for multiple different mice?

### DISCUSSION THREAD

**Discussion by jessebrennan** [noah-nuebling/mac-mouse-fix#563] [Questions]
I use two different mice at different work stations that I like to map slightly differently. Is there a way to save configurations for Mac Mouse Fix per-device?

**Comment by lordboncek** [comment]
I have the same problem - I am using a Razer Deathadder v3 and a Wacom Intuos Pro. I would like to have some features enabled on the Deathadder and disabled on the Wacom (e.g., reverse scrolling - it messes up panning with the Intuos). Is there any way to do this?

**Comment by hernaneche** [comment]
I'm also using two mice simultaneously, but I'm unable to configure both.

**Comment by geekscrapy** [comment]
+1 one mouse for work, one for play

**Reply by geekscrapy** [reply]
Logitech MX 3 and 4, FWIW
